src/components/Requirements/MultiplePlansDropdown.vue

```diff
@@ -37,14 +37,15 @@
 <script lang="ts">
 import { defineComponent } from 'vue';
 import editPlan from '@/assets/images/editplan.svg';
+import store from '@/store';
 
 export default defineComponent({
   props: {
     plan: { type: String, default: 'No additional plans yet' },
   },
   data() {
     return {
-      plans: ['PLAN 1', 'PLAN 2', 'PLAN 3', 'PLAN 4'],
+      plans: store.state.plans.map(plan => plan.name),
```

Comment:
Now that the user's plans are maintained in Firestore, we should probably make `plans` a computed property instead. Right now, deleting or adding a plan would not be reflected by the dropdown.

Reply:
Zak's a Vue wizard 🧙🏻‍♂️